meta-lmp-bsp/recipes-bsp/trusted-firmware-a/tf-a-stm32mp/0001-stm32mp1-save-boot-auth-status-and-partition-info.patch

```diff
@@ -0,0 +1,103 @@
+From fa9aaf082327709b2acfcae18358fac482834571 Mon Sep 17 00:00:00 2001
+From: Igor Opaniuk <igor.opaniuk@foundries.io>
+Date: Thu, 23 Jun 2022 15:30:12 +0300
+Subject: [PATCH] stm32mp1: save boot auth status and partition info
+
+Introduce a functionality for saving/restoring boot auth status and
+partition used for booting (FSBL partition on which the boot was successful).
+test
+
```

Comment:
Please add the upstream status for this patch.

E.g.:
`Upstream-Status: Pending`